src/scripts/feature-flags.js

```diff
@@ -1,20 +1,14 @@
 const featureFlag = {
   isExtension: window.chrome && window.chrome.extension,
-  // Tags are off in release right now
   tagsEnabled: true,
   compareEnabled: true,
   vendorsEnabled: true,
   qualityEnabled: true,
-  // Additional debugging / item info tools
-  debugMode: false,
-  // Print debug info to console about item moves
-  debugMoves: false,
-  // show changelog toaster
-  changelogToaster: $DIM_FLAVOR === 'release' || $DIM_FLAVOR === 'beta',
-
-  materialsExchangeEnabled: false,
-  // allow importing and exporting your DIM data to JSON
-  importExport: $DIM_FLAVOR !== 'release'
+  debugMode: false,                           // Additional debugging / item info tools
+  debugMoves: false,                          // Print debug info to console about item moves
+  changelogToaster: $DIM_FLAVOR !== 'dev',    // show changelog toaster
+  materialsExchangeEnabled: $DIM_FLAVOR === 'dev',
```

Comment:
Just to be super clear - I think until the materials exchange feature can be completely redesigned, it should stay 100% disabled.